src/main/java/hudson/plugins/sloccount/SloccountBuildAction.java

```diff
@@ -1,29 +1,37 @@
 package hudson.plugins.sloccount;
 
-import hudson.model.AbstractBuild;
+import hudson.model.Run;
 import hudson.model.Action;
+import jenkins.model.RunAction2;
 import hudson.plugins.sloccount.model.SloccountReportStatistics;
+import jenkins.tasks.SimpleBuildStep;
 
-import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
 
 import org.kohsuke.stapler.StaplerProxy;
 
 /**
  *
  * @author lordofthepigs
  */
-public class SloccountBuildAction implements Action, Serializable, StaplerProxy {
+public class SloccountBuildAction implements RunAction2, StaplerProxy, SimpleBuildStep.LastBuildAction  {
     /** Serial version UID. */
-    private static final long serialVersionUID = 0L;
```

Comment:
the comment above should be removed as well then. I would recommend double checking that serialization is not required anywhere by this plugin. But I would not expect Actions to be really serializable